### PR TITLE
Update hotkeys.js

### PR DESCRIPTION
**Pull request by StupidPz**
Add some numpad keys

### CODE REVIEW

File: dist/hotkeys.js [hunk 1]
```diff
@@ -98,7 +98,27 @@
     '\'': 222,
     '[': 219,
     ']': 221,
-    '\\': 220
+    '\\': 220,
+     isff ? 59 : 186,
+    '\'': 222,
+    '[': 219,
+    ']': 221,
+    '\\': 220,
```
Comment (jaywcjlove) [inline dist/hotkeys.js]: repeated  `'\\': 220,`

File: dist/hotkeys.js [hunk 2]
```diff
@@ -98,7 +98,27 @@
     '\'': 222,
     '[': 219,
     ']': 221,
-    '\\': 220
+    '\\': 220,
+     isff ? 59 : 186,
```
Comment (jaywcjlove) [inline dist/hotkeys.js]: Here is a failing.  @StupidPz

Comment (StupidPz) [reply dist/hotkeys.js]: should be ok now,try it again.

**Comment by jaywcjlove** [comment]
This is the document I found. Why add the `_` prefix?

https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code

<table>
<thead>
<tr>
  <th align="left">button</th>
  <th align="left">KeyCode</th>
  <th align="left">button</th>
  <th align="left">KeyCode</th>
  <th align="left">button</th>
  <th align="left">KeyCode</th>
</tr>
</thead>
<tbody><tr>
  <td align="left">/</td>
  <td align="left">111</td>
  <td align="left">*</td>
  <td align="left">106</td>
  <td align="left">-</td>
  <td align="left">109</td>
</tr>
<tr>
  <td align="left">+</td>
  <td align="left">107</td>
  <td align="left">delete</td>
  <td align="left">110</td>
  <td align="left">0</td>
  <td align="left">96</td>
</tr>
<tr>
  <td align="left">1</td>
  <td align="left">97</td>
  <td align="left">2</td>
  <td align="left">98</td>
  <td align="left">3</td>
  <td align="left">99</td>
</tr>
<tr>
  <td align="left">4</td>
  <td align="left">100</td>
  <td align="left">5</td>
  <td align="left">101</td>
  <td align="left">6</td>
  <td align="left">102</td>
</tr>
<tr>
  <td align="left">7</td>
  <td align="left">103</td>
  <td align="left">8</td>
  <td align="left">104</td>
  <td align="left">9</td>
  <td align="left">105</td>
</tr>
</tbody></table>

**Comment by StupidPz** [comment]
因为我要区分小键盘内的数字键和86键盘内的数字键，不然的话hotkeys的监听不好写，比方说我要监听小键盘中的数字7键，而不监听86键盘中的数字7键，又或者我要监听辅助键+小键盘的7比方说
```
hotkeys(shiftKey + '_7', function (event, handler) {
    event.preventDefault();
    console.log("press key " + shiftKey + '_7');
});
```
@jaywcjlove

**Comment by jaywcjlove** [comment]
@StupidPz  可以通过返回的 `key` 判断是不是 小键盘，或者提供个方法 `isSmallKeypad`，加下划线，并不能通过返回结果判断是不是小键盘，下划线也的方式设计方式不是很好。

```js
hotkeys('*', function(e){
  if(e.isSmallKeypad) {
     console.log('这是一个小键盘上的键!')
  }
});
```

File: dist/hotkeys.js [hunk 3]
```diff
@@ -98,7 +98,22 @@
...
-    '\\': 220
+    '\\': 220,
+	  '_0':96,
+	  '_1':97,
+	  '_2':98,
+	  '_3':99,
+	  '_4':100,
+	  '_5':101,
+	  '_6':102,
+	  '_7':103,
```
Comment (jaywcjlove) [inline dist/hotkeys.js]: image

image

image

https://developer.mozilla.org/zh-CN/docs/Web/API/KeyboardEvent/code

0x0047 | "Numpad7" |  71  
0x0009 | "Digit7" | 9   




@StupidPz

Comment (StupidPz) [reply dist/hotkeys.js]: got it，i'll try again

Comment (jaywcjlove) [reply dist/hotkeys.js]: @StupidPz 我的意思是你的键值好像不正确的样子。

**Pull request closed without merging**